.github/workflows/build-test-publish-package.yml

```diff
@@ -81,14 +81,23 @@ jobs:
           VAULT_VERSION: ${{ inputs.vault_version }}
           PVAULT_SERVICE_LICENSE: ${{ secrets.PVAULT_SERVICE_LICENSE }}
 
-      - name: Publish package
+      - name: Configure aws credentials
+        if: inputs.bump != 'none' && github.ref_name == 'main'
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::032109477184:role/vault-saas-prod-piiano-vault-typescript-github-actions
```

Comment:
Why are you assuming this role?

Reply:
To push the forms code to s3 of the CDN.
You can look at it `package.json` `push` command.